variables.tf

```diff
@@ -41,6 +41,27 @@ variable "allowed_cidr_blocks" {
   description = "List of CIDR blocks to be allowed to connect to the worker nodes"
 }
 
+variable "metadata_http_endpoint_enabled" {
```

Comment:
these new vars are not added to the `cloudposse/ec2-autoscale-group/aws` module?